src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/SearchServiceIntTest.kt

```diff
@@ -115,7 +115,6 @@ class SearchServiceIntTest : IntegrationTestBase() {
       ),
     )
 
-    stubOneHighConfidenceMatch()
```

Comment:
never called, an advantage of moving the stub verification to the lowest level was finding this unnecessary stub